app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,16 @@
+class LikesController < ApplicationController
+  def create
+    @user = current_user
+    @post = Post.find(params[:post_id] || params[:id])
```

Comment:
- [ ] Kindly add seed data to your schema, so that upon initialization, the forms can be availabe to the user